net/unison/Portfile

```diff
@@ -16,39 +15,41 @@ long_description    Unison is a file-synchronization tool for Unix and \
                     modified separately, and then brought up to date by \
                     propagating the changes in each replica to the other.
 homepage            http://www.cis.upenn.edu/~bcpierce/unison/
-master_sites        ${homepage}download/releases/${name}-${version}/
-platforms           darwin openbsd
+platforms           darwin
 
-set branch          [join [lrange [split ${version} .] 0 1] .]
+checksums           rmd160  24050d73c3d38389816828dd8ce77b4de7c80639 \
+                    sha256  8e071e862fef675ea034f0c3e4b881d392be0889db4f04ad894cc5a931891e3d \
+                    size    1400793
 
-checksums           rmd160  d5a1d66caffc02832c446f375a16319779fc5e0b \
-                    sha256  30aa53cd671d673580104f04be3cf81ac1e20a2e8baaf7274498739d59e99de8
+# from ocaml port
+supported_archs     i386 x86_64
+universal_variant   no
 
-patchfiles-append   patch-Makefile.diff
-platform darwin 8 {
-    patchfiles-append patch-Makefile.OCaml-MINOSXVERSION.diff
-}
-
-worksrcdir          src
+depends_build       bin:ocamlopt:ocaml
```

Comment:
I would just depend on the ocaml package. It isn't like we have incompatible packages around.

Reply:
You make a good point.
I was jus following what came before (``depends_build  bin:camlp4:ocaml``).
I have made the requested change.